etc/nebula-metad.conf.production

```diff
@@ -31,9 +31,3 @@
 ########## storage ##########
 # Root data path, here should be only single path for metad
 --data_path=data/meta
-
```

Comment:
Sorry for I miss the `heartbeat_interval_secs` in `nebula-storaged.conf.production`, please also add heartbeat_interval_secs=10 here, it need to be matched in two configuration. 

That's why the `show hosts` would show different things over different times.